i18n/en.json

```diff
@@ -6548,6 +6548,38 @@
     "translation": "Unable to load mattermost configuration file:  Adding DefaultClientLocale to AvailableLocales."
   },
   {
+    "id": "utils.config.bucket_empty.app_error",
+    "translation": "Failed: S3 bucket empty."
+  },
+  {
+    "id": "utils.config.error_checking_bucket_exist.app_error",
+    "translation": "Error checking if bucket exist."
```

Comment:
When does this error come up?

Reply:
Might occur when the bucket is not yet created or the bucket cannot be found on the predefined endpoint